openfecwebapp/templates/partials/reports-filter.html

```diff
@@ -31,19 +31,33 @@
   </div>
   <button type="button" class="js-accordion-trigger accordion__button">Report details</button>
   <div class="accordion__content">
-    <fieldset class="filter js-filter" data-filter="checkbox">
-      <legend class="label">Report status</legend>
-      <ul>
-        <li>
-          <input id="is_amended_true" name="is_amended" type="checkbox" value="true">
-          <label for="is_amended_true">Amended report</label>
-        </li>
-        <li>
-          <input id="is_amended_false" name="is_amended" type="checkbox" value="false">
-          <label for="is_amended_false">Non-amended report</label>
-        </li>
-      </ul>
-    </fieldset>
+    <div class="filter">
+      <fieldset class="js-filter js-filter-control" data-filter="checkbox" data-modifies-filter="most_recent" data-modifies-property="dataset">
+        <legend class="label">Version</legend>
+        <ul>
+          <li>
+            <input id="most_recent_true" name="most_recent" type="checkbox" value="true">
+            <label for="most_recent_true">Most recent version</label>
+          </li>
+        </ul>
+      </fieldset>
+    </div>
+
+    <div class="filter">
+      <fieldset class="js-filter js-filter-control" data-filter="checkbox" data-modifies-filter="amendment_indicator" data-modifies-property="dataset">
+        <legend class="label">Document status</legend>
+        <ul>
+          <li>
+            <input id="amendment_indicator_original" name="amendment_indicator" type="checkbox" value="N">
```

Comment:
I noticed that this filter doesn't seem to be supported by the API. Should check with @jontours .